src/App.js

```diff
@@ -1,8 +1,7 @@
 import React from "react";
-import { Routes } from "./routes";
+import AppRoutes from "./components/Routes";
 
 function App() {
-  return <Routes />;
+  return <AppRoutes />;
```

Comment:
Revert this to the previous state:

```suggestion
  return <Routes />;
```